src/main/java/pvpmode/internal/server/PvPServerEventHandler.java

```diff
@@ -42,12 +43,18 @@ public PvPServerEventHandler ()
     }
 
     /**
-     * Cancels combat events associated with PvP-disabled players. Note that this
-     * function will be invoked twice per attack - this is because of a Forge bug.
+     * Cancels combat events associated with PvP-disabled players. Note that
+     * this function will be invoked twice per attack - this is because of a
+     * Forge bug, but the {@link PvPCommonUtils#isCurrentAttackDuplicate()} call
+     * checks and returns if this call is a duplicate
      */
     @SubscribeEvent
     public void interceptPvP (LivingAttackEvent event)
     {
+        // This should alleviate duplicate entries.
```

Comment:
```suggestion
        // This alleviates duplicate entries.
```
This is becoming nitpicking at its finest, but documentation should not contain passive or uncertain language. After this change, you are free to merge. (I will thus already approve it, since I am boarding my plane soon.)